### ISSUE TITLE
Having multiple ChurchEventsList snippet calls in the same page

### ISSUE THREAD

**Issue by momon** [jgulledge19/Church-Events-Calendar#8]
There is an issue when there is more than one ChurchEventsList snippet calls in the same page.

In my setup I have a widget on the footer of every page which list the next 5 events using the ChurchEventsList.

This works fine, but if I go into the page which contains the calendar which has the snippet call: ChurchEventsCalendar and I do a filter of the events using the form, the page doesn't output the results within the page body itself but it goes to the bottom widget instead where I have the ChurchEventsList snippet call.

So basically the last ChurchEventsList snippet call is the one outputting the results.

I would suspect that ChurchEventsList needs to have a unique ID which you can specify on ChurchEventsCalendar so that results when filtering are linked properly.

I hope that wasn't too confusing.
